app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,25 @@
+class StatsController < ApplicationController
+  before_action :authorize_access
+
+  def new
+    @model = StatRange.new
+  end
+
+  def create
```

Comment:
This action does not behave like `create`,
Should we use an `index`? 🤔   (which is used to list and search resources, as happens in this case)